sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorExporterHelperExtensions.cs

```diff
@@ -18,6 +18,18 @@ public static class AzureMonitorExporterHelperExtensions
         /// <param name="configure">Exporter configuration options.</param>
         /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
         public static TracerProviderBuilder AddAzureMonitorTraceExporter(this TracerProviderBuilder builder, Action<AzureMonitorExporterOptions> configure = null)
+        {
+            return AddAzureMonitorTraceExporter(builder, out _, configure);
+        }
+
+        /// <summary>
+        /// Registers an Azure Monitor trace exporter that will receive <see cref="System.Diagnostics.Activity"/> instances.
+        /// </summary>
+        /// <param name="builder"><see cref="TracerProviderBuilder"/> builder to use.</param>
+        /// <param name="configure">Exporter configuration options.</param>
+        /// <param name="processor">WORK IN PROGRESS.</param>
+        /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+        public static TracerProviderBuilder AddAzureMonitorTraceExporter(this TracerProviderBuilder builder, out ActivityProcessor processor, Action<AzureMonitorExporterOptions> configure)
```

Comment:
Why do we want this in the function signature?

Reply:
To be honest, I don't :)
But i couldn't find any public APIs get a handle on the ActivityProcessor.
I can get a handle on the `TracerProviderBuilder`, but it's processor is private.

This part is a work in progress, proving that calling ForceFlush works.

Reply:
var exporter = new AzureMonitorTraceExporter(...);
var processor = new BatchExportActivityProcessor(exporter);

builder.AddProcessor(processor).Build();

// do something

var result = processor.ForceFlush(timeout);

// do something with result